cli/script/command-executor.ts

```diff
@@ -59,8 +62,12 @@ export interface PackageWithMetrics {
     metrics?: UpdateMetricsWithTotalActive;
 }
 
-export var sdk: AccountManager;
 export var log = (message: string | Chalk.ChalkChain): void => console.log(message);
+export var sdk: AccountManager;
+export var spawn = childProcess.spawn;
+export var getTmpDir = (): string => {
```

Comment:
Actually.. maybe its not necessary to stub this. I was initially worried that os.tmpdir() might not return a constant value.